README.md

```diff
@@ -246,12 +250,13 @@ Start autocannon against the given target.
     * `ignoreCoordinatedOmission`: A `Boolean` which disable the correction of latencies to compensate the coordinated omission issue. Does not make sense when no rate of requests has been specified (`connectionRate` or `overallRate`). _OPTIONAL_ default: `false`.
     * `reconnectRate`: A `Number` which makes the individual connections disconnect and reconnect to the server whenever it has sent that number of requests. _OPTIONAL_
     * `requests`: An `Array` of `Object`s which represents the sequence of requests to make while benchmarking. Can be used in conjunction with the `body`, `headers` and `method` params above. Check the samples folder for an example of how this might be used. _OPTIONAL_. Contained objects can have these attributes:
-       * `body`: When present, will override `opts.body`. _OPTIONAL_.
-       * `headers`: When present, will override `opts.headers`. _OPTIONAL_.
-       * `method`: When present, will override `opts.method`. _OPTIONAL_.
-       * `path`: When present, will override `opts.path`. _OPTIONAL_.
-       * `setupRequest`: A `Function` you may provide to mutate the raw `request` object, e.g. `request.method = 'GET'`. It takes `request` (Object) and `context` (Object) parameters, and must return the modified request. When it returns a falsey value, autocannon will restart from first request. _OPTIONAL_.
-       * `onResponse`: A `Function` you may provide to process the received response. It takes `status` (Number), `body` (String) and `context` (Object) parameters. _OPTIONAL_.
+       * `body`: When present, will override `opts.body`. _OPTIONAL_
+       * `headers`: When present, will override `opts.headers`. _OPTIONAL_
+       * `method`: When present, will override `opts.method`. _OPTIONAL_
+       * `path`: When present, will override `opts.path`. _OPTIONAL_
+       * `setupRequest`: A `Function` you may provide to mutate the raw `request` object, e.g. `request.method = 'GET'`. It takes `request` (Object) and `context` (Object) parameters, and must return the modified request. When it returns a falsey value, autocannon will restart from first request. _OPTIONAL_
+       * `onResponse`: A `Function` you may provide to process the received response. It takes `status` (Number), `body` (String) and `context` (Object) parameters. _OPTIONAL_
+    * `har`: the `String` location of an [HAR](https://w3c.github.io/web-performance/specs/HAR/Overview.html) file. Autocannon will extract request from this file, keeping only the ones targeting the same domains as the given urls. Requests method, path, query parameters, headers and body will be set from the HAR file. _NOTE_: this will take over `requests` property. You can add extra headers with `headers` property, but `method`, `form` and `body` will be ignored. _OPTIONAL_
```

Comment:
I think it's a path on disk. It would be also ok to support it as a Javascript object.

Reply:
Correct, it's the path on disk. I'll change the comment.
About JSON content: do you want to support both, or only JSON?

Reply:
I would just support a JS object in HAR format and leave the parsing for the CLI.